install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,147 @@ Check VNIDs by running:
+ plugin_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/vsp-openshift-4.0.X1830.el7.centos.x86_64.rpm
+
+ # Optional parameters
+ nuage_interface_mtu=1460
+ nuage_master_adminusername=admin
+ nuage_master_adminuserpasswd=admin
+ nuage_master_cspadminpasswd=admin
+ nuage_openshift_monitor_log_dir=/var/log/nuage-openshift-monitor
+
+ # Required for brownfield install (where a {product-title} cluster exists without Nuage as the networking plugin)
+ nuage_dockker_bridge=lbr0
+----
+====
+
+==== Installation for multiple masters (HA)
+
+A High Availability (HA) environment can be configured with multiple masters and multiple nodes. There are two supported configuration methods for HA--native HA configuration and configuration using Pacemaker.
+
+Nuage VSP integration in multi-master mode only supports the native HA configuration method described in this section. This can be combined with any load balancing solution, the default being HAProxy. In the inventory file, there are three master hosts, the nodes, an etcd server and a host that functions as the HAProxy to balance the master API on all master hosts. The HAProxy host is defined in the [lb] section of the inventory file enabling Ansible to automatically install and configure HAProxy as the load balancing solution. In the Ansible nodes file, the following parameters need to be specified in order to setup Nuage VSP as the network plugin:
+
+====
+----
+ # Create and OSEv3 group that contains masters, nodes, load-balancers, and etcd hosts
+ masters
+ nodes
+ etcd
+ lb
+
+ # Nuage specific parameters
+ openshift_use_openshift_sdn=False
+ openshift_use_nuage=True
+ os_sdn_network_plugin_name='nuage/vsp-openshift'
+ openshift_node_proxy_mode='userspace'
+
+ # VSP related parameters
+ vsd_api_url=https://192.168.103.200:8443
+ vsp_version=v4_0
+ enterprise=nuage
+ domain=openshift
+ vsc_active_ip=192.168.103.201
+ vsc_standby_ip=192.168.103.202
+ uplink_interface=eth0
+
+ # rpm locations
+ nuage_openshift_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/nuage-openshift-monitor-4.0.X.1830.el7.centos.x86_64.rpm
+ vrs_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/nuage-openvswitch-4.0.X.225.el7.x86_64.rpm
+ plugin_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/vsp-openshift-4.0.X1830.el7.centos.x86_64.rpm
+
+ # Required for Nuage Monitor REST server and HA
+ openshift_master_cluster_method=native
+ openshift_master_cluster_hostname=lb.nuageopenshift.com
+ openshift_master_cluster_public_hostname=lb.nuageopenshift.com
+ nuage_openshift_monitor_rest_server_port=9443
+ 
+ # Optional parameters
+ nuage_interface_mtu=1460
+ nuage_master_adminusername='admin's user-name'
+ nuage_master_adminuserpasswd='admin's password'
+ nuage_master_cspadminpasswd='csp admin password'
+ nuage_openshift_monitor_log_dir=/var/log/nuage-openshift-monitor
+
+ # Required for brownfield install (where a {product-title} cluster exists without Nuage as the networking plugin)
```

Comment:
Same comment here as before. the {product-title} macro doesn't expand in a code snippet.